docs/API/header.md

```diff
@@ -62,17 +62,19 @@ Components defined through props take precedence over components passed in as ch
 
 ### Header customisability
 
-We wanted the Header to be as customisable as possible, so you are free to try different combinations of props.
+We wanted the Header to be as customisable as possible, so you are free to try different combinations of props. For example if you want to change left,center,right component's layout, try to play with ` innerContainerStyles `
```

Comment:
Maybe something more friendly?

For example, if you want to change the left, center, or right component's layout, you can adjust the `innerContainerStyles`